wrapper/CSharp/wolfSSL_CSharp/X509.cs

```diff
@@ -19,10 +19,22 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
 
+
+/* CE Not always reliably detected. Define your own WindowsCE as needed */
+#if _WIN32_WCE || WINCE || PocketPC
+    #if !WindowsCE

```

Comment:
Please don't raise a warning here, just force define `WindowsCE` and remove your NEED_WINDOWS_CE and warnings.

```
#undef WindowsCE
#define WindowsCE
```